src/main/java/edu/iis/powp/factory/MyFigureJoe1Factory.java

```diff
@@ -0,0 +1,23 @@
+package edu.iis.powp.factory;
+
+import edu.iis.powp.command.CommandDrawLineToPosition;
+import edu.iis.powp.command.CommandSetPosition;
+import edu.iis.powp.command.ComplexCommand;
+import edu.iis.powp.command.PlotterCommand;
+
+public class MyFigureJoe1Factory {
+	public static PlotterCommand prepareCommand(int x, int y, int sizeX) {
```

Comment:
The solution was supposed to be generic and work with any script written in similar way as FigureJoe scripts.